torch/_inductor/lowering.py

```diff
@@ -5339,6 +5342,35 @@ def mul(a, b):
         return make_pointwise(fn)(a, b)
 
 
+def get_constant_value(x: ir.IRNode) -> Optional[ir.Constant]:
+    """Try convert an arbitrary IR node into an ir.Constant value"""
+    if isinstance(x, ir.MutableBox):
+        return get_constant_value(x.data)
+    if isinstance(x, ir.BaseView):
+        return get_constant_value(x.unwrap_view())
```

Comment:
Is this just an optimisation? Could we let the code below handle these two cases?

Reply:
Yeah, I could just drop these and put everything through `ExtractConstantsHandler` but this avoids creating all the intermediate sympy indexing expressions if they aren't required.

Reply:
Mind leaving a comment?